src/Avalonia.Labs.Controls/Cache/ImageCache.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Avalonia.Media.Imaging;
+
+namespace Avalonia.Labs.Controls.Cache
+{
+    /// <summary>
+    /// Provides methods and tools to cache images in a folder
+    /// </summary>
+    public class ImageCache : CacheBase<Bitmap>
```

Comment:
I would keep it internal for now

Reply:
This and related classes are now internal

Reply:
what about InMemoryStorage?